core/build.gradle

```diff
@@ -37,7 +37,7 @@ tasks.register('versionWriterTask') {
     }
     it.inputs.property("versionName", versionName)
 
-    def generatedDir = project.layout.buildDirectory.dir("generated/version")
+    def generatedDir = project.layout.buildDirectory.dir("generated_stone_source/main/src")
```

Comment:
![Screenshot 2024-03-13 at 7 00 22 PM](https://github.com/dropbox/dropbox-sdk-java/assets/3921079/15033b25-84df-43c3-aada-ac1daa604809)

This consolidates the generated build source path into one dir - which fixes a bug in the IDE which wouldnt index these files properly, though assemble worked fine since these files are still on the the classpath